org.eclipse.xtext.ide.tests/src/org/eclipse/xtext/ide/tests/server/UriExtensionsTest.xtend

```diff
@@ -276,6 +276,20 @@ class UriExtensionsTest {
 		var uri = createFileURI(directory.absolutePath + "/").toUriString.toUri
 		assertTrue(uri.isPrefix)
 	}
+	
+	@Test
+	def void test_withEmptyAuthority () {
+		val uriWithoutAuthority = org.eclipse.emf.common.util.URI.createURI("file:/path/to/resource/")
```

Comment:
Any reason you don't just write "URI" (without the FQN) here?

Reply:
Name clash?

Reply:
Yes, net.URI is already imported